### PR TITLE
Repaired luigi build

### PR DESCRIPTION
**Pull request by Aaronhrndz**
## Description
Changed URI.js route

## Motivation and Context
Luigi does not build properly, referencing issue #2028

## Have you tested this? If so, how?
The project has passed the travis tests, luigid is running properly.

### CODE REVIEW

File: luigi/static/visualiser/index.html [hunk 1]
```diff
@@ -21,11 +21,10 @@
         <script src="lib/jquery.slimscroll.min.js"></script>
         <script src="lib/AdminLTE/js/app.min.js"></script>
         <script src="lib/datatables/js/jquery.dataTables.min.js"></script>
-        <script src="lib/URI.js/1.18.2/URI.js"></script>
+        <script src="lib/URI.js"></script>
```
Comment (Tarrasch) [inline luigi/static/visualiser/index.html]: Wasn't the issue the `URI.js` matching some regex? Do we not have the same issue here?

Comment (Aaronhrndz) [inline luigi/static/visualiser/index.html]: In setup.py it copies the static files with this function:

def get_static_files(path):
    return [os.path.join(dirpath.replace("luigi/", ""), ext)
            for (dirpath, dirnames, filenames) in os.walk(path)
            for ext in ["*.html", "*.js", "*.css", "*.png",
                        "*.eot", "*.svg", "*.ttf", "*.woff", "*.woff2"]]


The URI folder was called URI.js, and the above function fails when it gets this folder extension

**Comment by dlstadther** [comment]
It's been requested and (somewhat) supported by @nmb10 and @kwilcox to leave the version number within the path. So just change `luigi/static/visualiser/lib/URI.js/1.18.2./URI.js` to `luigi/static/visualiser/lib/URI/1.18.2/URI.js`.

File: luigi/static/visualiser/index.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
         <script src="lib/jquery.slimscroll.min.js"></script>
         <script src="lib/AdminLTE/js/app.min.js"></script>
         <script src="lib/datatables/js/jquery.dataTables.min.js"></script>
-        <script src="lib/URI.js"></script>
+        <script src="lib//URI/1.18.2/URI.js"></script>
```
Comment (dlstadther) [inline luigi/static/visualiser/index.html]: `lib/URI/1.18.2/URI.js`

one too many `/`

**Comment by dlstadther** [comment]
@Tarrasch @nmb10 Are you good with this?

**Comment by nmb10** [comment]
Hi, @dlstadther . Yes.

**Comment by miku** [comment]
> Can anyone else verify this works?

I believe this PR would fix the installation problem, that I encountered:

```
$ pip install luigi
....
error: can't copy 'luigi/static/visualiser/lib/URI.js': doesn't exist or not a regular file
```

In setup.py, [get_static_files](https://github.com/spotify/luigi/blob/7701d9fb4ad2ec859d208fde9fd475bbc43c64a1/setup.py#L20) will return (among other globs) `'static/visualiser/lib/*.js',`, which will match [URI.js](https://github.com/spotify/luigi/tree/master/luigi/static/visualiser/lib/URI.js), which is not a regular file, as setup seems to expect.

**Comment by dlstadther** [comment]
Thanks!

**Comment by Lucas-C** [comment]
I have the same error when `pip install`ing luigi 2.6.0. I cannot use this version in its current state.
Could you please include this fix in a 2.6.1 release ?

**Comment by dlstadther** [comment]
@Lucas-C I don't have the ability to push luigi releases. That'll have to wait until @Tarrasch returns.

**Comment by shahabedinh** [comment]
I had issue 

/opt/cloudera/parcels/Anaconda-4.0.0/bin/pip install luigi

....
reating build/lib/luigi/static/visualiser/lib
    copying luigi/static/visualiser/lib/mustache.js -> build/lib/luigi/static/visualiser/lib
    copying luigi/static/visualiser/lib/jquery-1.10.0.min.js -> build/lib/luigi/static/visualiser/lib
    error: can't copy 'luigi/static/visualiser/lib/URI.js': doesn't exist or not a regular file


then I download and then pip it.  

wget https://github.com/spotify/luigi/archive/master.zip
unzip master.zip
cd luigi-master/
sudo /opt/cloudera/parcels/Anaconda/bin/pip install .

it works for me

**Pull request merged**